CHANGELOG.rst

```diff
@@ -17,6 +17,10 @@ Unreleased
 ----------
 None
 
+[3.50.0]
+--------
+refactor: remove enterprise_catalogs API endpoint and EnterpriseApiClient with usages
```

Comment:
nit: 
```suggestion
refactor: remove enterprise_catalogs API endpoints and EnterpriseApiClient
```

Reply:
I removed the only one endpoint `enterprise_catalogs`

Reply:
missed that, lets's have these details in the CHANGELOG then

Reply:
updated to `refactor: remove enterprise_catalogs API endpoint and EnterpriseApiClient`